python/paddle/v2/framework/tests/test_modified_huber_loss_op.py

```diff
@@ -32,7 +41,7 @@ def test_check_output(self):
         self.check_output()
 
     def test_check_grad(self):
-        self.check_grad(['X'], 'Out', max_relative_error=0.005)
+        self.check_grad(['X'], 'Out', max_relative_error=0.01)
```

Comment:
Just a random question: could we have a systematic way to set the error tolerance?

Reply:
There are a lot of indifferentiable points for forwarding operators. Only writers of operators could know which point is indifferentiable.

So I think that should be managed manually by Op writers.